conf/application.conf

```diff
@@ -219,6 +219,7 @@ play.filters {
 
   # Disabled filters remove elements from the enabled list.
   #disabled += filters.ExampleFilters
+  disabled += play.filters.csrf.CSRFFilter
```

Comment:
why did you disable the csrf filters? this is an essential security feature which we definitively don't want to disable.

Reply:
this function was disabled when we developed post methods. This filter poses a problem with the post method (403 error).

Reply:
You need to correctly configure the filter to allow your explicit request. As stated above removing a security feature because it causes you a problem is no solution, please fix this.